func_tests/test_client.py

```diff
@@ -36,3 +38,28 @@ def test_record_sets(vinyldns_test_context):
     assert change.record_set.name == rs.name
     assert all([l.__dict__ == r.__dict__ for l, r in zip(change.record_set.records, rs.records)])
 
+
+def test_batch_change(vinyldns_test_context):
+    changes = [
+        AddRecord('change-test.vinyldns.', RecordType.A, 200, AData('192.0.2.111')),
+        AddRecord('192.0.2.111', RecordType.PTR, 200, PTRData('change-test.vinyldns.'))
```

Comment:
this is the thing that was broken.  Before, instead of passing in `AData`, we were passing in just `1.2.3.4` which was silly.  Surprisingly the json stuff still passed, which is sad.

Updated the tests to pass in real RData sections